src/components/UserInfo.vue

```diff
@@ -220,7 +219,15 @@ export default {
   data() {
     return {
       maxLength: 250,
-      userStats: null,
+      userStats: {
```

Comment:
Default values for user stats ensures the whole component will be visualized even if the data is delayed.